src/esp/.gitignore

```diff
@@ -1,2 +1,3 @@
 build/
+sdkconfig
```

Comment:
Interesting. Why did you add sdkconfig to .gitignore? Wouldn't it be easier for a developer not having to configure the ESP? There are quite a few parameters that you would have to set manually and carefully document.

Reply:
That file is generated when you do the initial build (based on interactive questions during build).  My build overwrote the one you had checked in so it seemed like you might not want to keep it in your repo.

Reply:
For questions that I answered during the initial config, I believe the defaults that were presented to you are taken from sdkconfig. If you remove that file, you will get the global defaults. Try this: remove sdkconfig then rebuild. When you go through the prompts make sure that the second core is activated and the watchdog timers are disabled (which were not the defaults when I did this for the first time). If the defaults are different, that would support my theory. In that case I opt for leaving sdkconfig in the repo.

Reply:
This is a little weird.  I deleted my local sdkconfig and sdkconfig.old files then did a "make clean" and "make" to rebuild the project.  It didn't ask the build questions nor did it generate a new sdkconfig file.  Not sure how or why.  For now I'll just add the sdkconfig file back and update the .gitignore.  I'll email you my build logs (initial and rebuild) in case you want to look at this a little more.  Wish me luck making my changes and squashing them.

Reply:
I was looking at the logs again.  It looks like it *does* generate a new sdkconfig when you do a "make clean".  Not sure what I was looking at before.  In any case, the sdkconfig I re-added was the one from your project (ie. the original one).  

Message shown during a "make clean":
DEFCONFIG
#
# configuration written to /Users/kpw/dev/git/RetroStoreCard/src/esp/sdkconfig